pam/integration-tests/vhs-helpers_test.go

```diff
@@ -144,6 +145,7 @@ func newTapeData(tapeName string, settings ...tapeSetting) tapeData {
 		vhsMargin:      0,
 		vhsShell:       "bash",
 		vhsWaitTimeout: 10 * time.Second,
+		vhsTypingSpeed: 5 * time.Millisecond,
```

Comment:
Please mention in the commit message what the previous typing speed was

Reply:
TODO: At rebase phase.

Reply:
It was 50ms, for reference:

https://github.com/charmbracelet/vhs/blob/28be2de8db81a1066271d99bc511046fa1f91da0/vhs.go#L59